thunder/torch/__init__.py

```diff
@@ -4552,6 +4555,12 @@ def interpolate(
 
     utils.check(a.ndim >= 3, lambda: f"Expected {a.ndim=} >= 3")
     utils.check(a.numel() > 0, lambda: f"Expected {a.numel=} to be greater than 0")
+    utils.check(align_corners == None, lambda: f"'align_corners' is not yet supported.")
```

Comment:
Nit: should this (and the following) be `exception_type=NotImplementedError` and maybe "not yet supported _in Thunder_"?

Reply:
From the doc: https://pytorch.org/docs/stable/generated/torch.ao.nn.quantized.functional.interpolate.html, `align_corners` has only effect when `mode = 'bilinear'`.

Reply:
didn't know that was an option, neat :-)
yes, it should be. will fix in a follow-up.

Reply:
#721